dotenv.example

```diff
@@ -5,7 +5,7 @@ SPARC_API_USERNAME=iyqu2bm3T4rDWWpnwYfEl2i6o2Sphpuq_TKvFnIOTA0
 SPARC_API_PASSWORD=qL1oZmdMi7OU7G1f0q6aCTVR6HrmoNmpZVNc8DfHqek
 FAYE_TOKEN=3bd16424a2154769a1fb4adc081b10747bdbfdbd93682032
 CWF_FAYE_HOST=localhost:9292
-DOCUMENT_ROOT=tmp/documents
+DOCUMENTS_FOLDER=documents
```

Comment:
@goleksiak I added "/documents" to .gitignore. The reason to move the documents folder is to get it out of the public directory, which was being used because DOCUMENT_ROOT was somehow being set to /public/ for controllers but not the importer jobs. Thus, we renamed the ENV variable to prevent clashing with environment variables being set by some combination of Rails, Passenger, and Apache.